sass/includes/_variables.scss

```diff
@@ -33,6 +33,7 @@ $color__blue_5: #8ec0ec;
 $color__purple: #4c2c92;
 $color__red: #a72916;
 
+$color__light-grey: #fafafa;
```

Comment:
This has been updated to #D9D9D6 to match the new brand book (https://www.figma.com/file/uo8YY98lLws8E1HEfpdn5X/TNA-Etna---Design-system?type=design&node-id=8%3A38&t=I6lb9pSno1ZGEVLp-1)

Reply:
Just checking the component design, you were right to use #fafafa but this maps to grey-100 in the design system, light-grey is a brand colour defined at #d9d9d6, which may confuse things… I guess we could do with updating the codebase to match Figma or vice-versa

Reply:
ah okay bad naming choice on my part! shall I update this to grey-100 instead in the variables file and use that?